File: src/main/java/me/kuku/yuq/service/BiliBiliService.java
package me.kuku.yuq.service;

import com.IceCreamQAQ.Yu.annotation.AutoBind;
import me.kuku.yuq.entity.BiliBiliEntity;

import java.util.List;

@AutoBind
public interface BiliBiliService {
    BiliBiliEntity findByQQ(Long qq);
    void save(BiliBiliEntity biliBiliEntity);
    int delByQQ(Long qq);
    List<BiliBiliEntity> findByMonitor(Boolean monitor);
    List<BiliBiliEntity> findAll();
    List<BiliBiliEntity> findByTask(Boolean task);
}

File: src/main/kotlin/me/kuku/yuq/service/BiliBiliService.kt
package me.kuku.yuq.service

import com.IceCreamQAQ.Yu.annotation.AutoBind
import me.kuku.yuq.entity.BiliBiliEntity

@AutoBind
interface BiliBiliService {
    fun findByQQ(qq: Long): BiliBiliEntity?
    fun save(biliEntity: BiliBiliEntity)
    fun delByQQ(qq: Long): Int
    fun findByMonitor(monitor: Boolean): List<BiliBiliEntity>
    fun findAll(): List<BiliBiliEntity>
}